commands/build.go

```diff
@@ -269,8 +269,7 @@ func (o *buildOptionsHash) String() string {
 }
 
 func runBuild(ctx context.Context, dockerCli command.Cli, options buildOptions) (err error) {
-	mp := dockerCli.MeterProvider(ctx)
-	defer metricutil.Shutdown(ctx, mp)
+	mp := dockerCli.MeterProvider()
```

Comment:
This returns a meter provider initialized by the CLI now and the CLI handles reporting and shutdown now.

Need to double check that this works correctly with standalone.